### DISCUSSION TITLE
Is there an overview regarding Repeaters V8 vs Repeaters V9

### DISCUSSION THREAD

**Discussion by klodoma** [directus/directus#10618] [Archive]
I am evaluating an upgrade from V8 to V9. We are heavily using the repeaters in V8 and I cannot find some of them in V9. Is there somehwere an overview or a comparison between the 2?

I'll just post here a few questions that popped up very fast:

1. How to delete/dupplicate a single repeated element in V9?
![image](https://user-images.githubusercontent.com/944350/146795921-04cb3a6e-87d9-454f-9b40-5657ad74779d.png)

2. How to set a default value for a repeated element in V9?
![image](https://user-images.githubusercontent.com/944350/146796146-731a5734-c27d-43ea-84ea-c43ba7a9cc38.png)

3. Is it possible to select a value from another collection withing a repeater field?

Thank you in advance.

**Comment by rijkvanzanten** [comment]
> How to delete/dupplicate a single repeated element in V9?
> How to set a default value for a repeated element in V9?

There's currently no options to duplicate/ set default values for the repeater interface. More than happy to review/merge a PR to add those though 🙂 

> Is it possible to select a value from another collection withing a repeater field?

No, repeaters are static JSON fields that can't have relationships to other collections. To achieve that, use a one-to-many or many-to-many field instead 👍🏻
